src/test/java/teammates/test/cases/automated/SubmissionsAdjustmentTest.java

```diff
@@ -32,6 +32,7 @@
 import teammates.logic.core.FeedbackResponsesLogic;
 import teammates.logic.core.FeedbackSessionsLogic;
 import teammates.logic.core.StudentsLogic;
+import teammates.test.cases.common.FieldValidatorTest;
```

Comment:
Doesn't make sense for a test case to be dependent on another (I specifically target this in #5570).